src/api-reference/financial-integration/v4.financial-integration.markdown

```diff
@@ -429,24 +607,44 @@ Code|Description|Category
 
```

Comment:
The schema names should match the names in Swagger- people will be looking for the exact strings returned by the API. So, these schemas should be renamed to match what is in Swagger.

Reply:
We were trying to see how to find the difference and unable to see what's wrong here. Can you please give one example.

Reply:
Examples sent in email.

Reply:
Updated the schema names to match the swagger.  A new PR is uploaded.